ci-operator/config/red-hat-data-services/opendatahub-operator/red-hat-data-services-opendatahub-operator-v1.0.0.yaml

```diff
@@ -0,0 +1,92 @@
+base_images:
+  os:
+    name: centos
+    namespace: origin
+    tag: "8"
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.14
+images:
+- context_dir: odh-manifests/tests/
+  dockerfile_path: Dockerfile
+  from: os
+  to: odh-tests
+- context_dir: .
+  dockerfile_path: build/Dockerfile.modh
+  from: os
+  to: opendatahub-operator
+operator:
+  bundles:
+  - dockerfile_path: Dockerfile.bundle
+  substitutions:
+  - pullspec: quay.io/opendatahub/opendatahub-operator:v0.9.0
```

Comment:
shouldn't we pull 1.0.0 now?

Reply:
This has to do with the bundle that is included in the repo.  It's only used in the test runs, so it hasn't been updated.  It's just gets replaced with the image for the operator that is built during this test run.